### PR TITLE
*: improve the some metrics

### PR DESCRIPTION
**Pull request by rleungx**
Signed-off-by: Ryan Leung <rleungx@gmail.com>

### What problem does this PR solve?

We have some monitors like the balance region scheduler which records the scheduling event. But it's hard to find something when we investigate the problem.

### What is changed and how it works?

This PR records the total number of events so that we can know the relationship between each event.

### Check List

Tests

- Unit test

### Release note

```release-note
None
```

### CODE REVIEW

File: server/schedulers/balance_region.go [hunk 1]
```diff
@@ -150,6 +150,7 @@ func (s *balanceRegionScheduler) Schedule(cluster opt.Cluster) []*operator.Opera
 	})
 	for _, plan.source = range stores {
 		for i := 0; i < balanceRegionRetryLimit; i++ {
+			schedulerCounter.WithLabelValues(s.GetName(), "total").Inc()
```
Comment (lhy1024) [inline server/schedulers/balance_region.go:153]: total? balance_total?

Comment (rleungx) [reply server/schedulers/balance_region.go:153]: IMO, total is enough ^_^

**Review by lhy1024 — APPROVED** [review]

**Review by nolouch — APPROVED** [review]
lgtm

**Comment by nolouch** [comment]
Do we need to cherry-pick?

**Comment by rleungx** [comment]
> Do we need to cherry-pick?

Since it's only a small improvement, I think we don't need to cherry-pick it.

**Pull request merged**